client/src/components/Auction.jsx

```diff
@@ -63,8 +63,9 @@ class Auction extends Component {
           } else {
             answer.json()
             .then((bid) => {
-              console.log(bid);
+              console.log(bid.current_bid);
               //dispatch(bids.toggleSend());
+              alert('you\'re currently winning this auction! The price is now: $' + bid.current_bid);
```

Comment:
I personally recommend the message on line 68 to be `You have successfully bid $ ${bid.current_bid}`, because when the user make a bid, for sure he/she is the winner at the moment.